frontend/src/modules/dataset/dataset-browser.tsx

```diff
@@ -207,7 +207,15 @@ export function DatasetBrowser (props: DatasetBrowserProps) {
                 <Condition condition={header !== undefined}>
                     { header }
                 </Condition>
-                
+
+                <Condition condition={manageMode ? [DatasetBrowserManageMode.Create, DatasetBrowserManageMode.Edit].includes(manageMode) : false}>
+                    <div style={{border: 'dashed 1px', padding: '15px'}}>
```

Comment:
If we're already adding a CSS file we should probably move this styling to there too.